test/parallel/test-net-server-options.js

```diff
@@ -1,9 +1,16 @@
 'use strict';
-require('../common');
 const assert = require('assert');
+const common = require('../common');
```

Comment:
Actually `require('../common');` [should be first](https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#lines-1-2). It does some wiring, and maybe we'll want it to monkeypatch `assert` in the future.